ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -228,7 +229,8 @@ export default class SignatureRequestOriginal extends Component {
               className="request-signature__help-link"
               onClick={() => {
                 global.platform.openTab({
-                  url: 'https://consensys.net/blog/metamask/the-seal-of-approval-know-what-youre-consenting-to-with-permissions-and-approvals-in-metamask/',
+                  url:
```

Comment:
My linter auto-changed this, I can revert if needed

Reply:
Hmm yup linter has an issue haha. Try `yarn lint:fix` should fix it

Reply:
Done!